.travis.compose.yml

```diff
@@ -4,8 +4,23 @@ services:
   test-hbmpc:
     build:
       context: .
-      dockerfile: .ci/Dockerfile
+      dockerfile: Dockerfile
       args:
         - BUILD
     volumes:
-      - .:/usr/src/HoneyBadgerMPC
+      - ./aws:/usr/src/HoneyBadgerMPC/aws
```

Comment:
I like this better since this wold keep the docker environment cleaner but we probably should document at some point that in case you add a folder at the root level then remember to update the dockerfile..

Reply:
This is important in order not to override the built `pypairing` binary.